### PR TITLE
Fix translations issues

### PR DESCRIPTION
**Pull request by imobachgs**
This PR should fix [bsc#928909](https://bugzilla.suse.com/show_bug.cgi?id=928909) and [bsc#928775](https://bugzilla.suse.com/show_bug.cgi?id=928775).

pot-files should be updated.

### CODE REVIEW

File: src/dialogs.ycp [hunk 1]
```diff
@@ -77,20 +77,20 @@ term EditDialogContents()
 					    `VBox(
 						  // combo box label: enable FCoE (yes/no)
 						  `ComboBox( `id(`fcoe), _("&FCoE Enable"),
-							     [`item(`id("yes"), "yes" ),
-							      `item(`id("no"), "no", true ) ]
+							     [`item(`id("yes"), _("Yes") ),
```
Comment (lslezak) [inline src/dialogs.ycp]: I have checked the [Label](https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/Label.rb) and the [Message](https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/Message.rb) and surprisingly we do not have such common texts defined. `Label.Yes/NoButton` contains the `&` keyboard shortcut marker and cannot be used here.

But I found [`Yast::String.YesNo`](https://github.com/yast/yast-yast2/blob/562e2f3ccfa8413a0528cf067ea7533fa72a2450/library/types/src/modules/String.rb#L99).

I'd suggest something like this:

```ruby
def yes_str
  Yast::String.YesNo(true)
end

def no_str
  Yast::String.YesNo(false)
end
...
[`item(`id("yes"), yest_str, ...]
```

Or you could even DRY it a bit more:

```ruby
def yesno_list(default)
  [`item(`id("yes"), yes_str, default == :yes ), `item(`id("no"), no_str, default == :no )]
end
...
ComboBox( `id(`fcoe), _("&FCoE Enable"), yesno_list(:no))
```

Comment (lslezak) [inline src/dialogs.ycp]: Ooooooh, it's YCP! :zap: :grin: Use YCP equivalent of the above code.

BTW does it actually make sense to fix translations for SLE11-SP3? I would doubt even about SP4...

Comment (imobachgs) [inline src/dialogs.ycp]: `String::YesNo` is not available in SLE 11 SP4. https://github.com/yast/yast-yast2/blob/Code-11-SP4/library/types/src/String.ycp

Comment (lslezak) [inline src/dialogs.ycp]: Oh, OK, I didn't know that is is missing in SP4... Yes, it does not make much sense adding it.

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**